Default message data populators

petringo opened the pull request:
Closes #13.

kasarit reviewed (dismissed):
Looks very good. I leave it to your consideration whether the `forceCustomMessageTypeLocationIndicatorTypes` flag in `MessageDataPopulator` is necessary. Removing it would lead to more verbose and explicit configuration needs, but at the same time the configuration could be somewhat simpler to understand.

petringo commented:
Thanks for lingual corrections! I simplified the configuration dropping the `forceCustomMessageTypeLocationIndicatorTypes` property.

Additionally, I fixed a (maybe theoretical) shortcoming, that when resolving the location indicator, message type was used from input message instead of most recently resolved type in builder. In general this makes difference only if message type is not available in input message.
